build.gradle

```diff
@@ -24,6 +24,7 @@ dependencies {
 
     testImplementation "org.junit.jupiter:junit-jupiter:$junit_version"
     testImplementation "io.mockk:mockk:$mockk_version"
+    testImplementation("io.strikt:strikt-jvm:0.32.0")
```

Comment:
We already use JUnit assertions. Is strikt needed here?

Reply:
It is more powerful than JUnit generally and also Kotlin-aware, but if you prefer JUnit style of assertion, I can drop it.